cognite/seismic/protos/v1/seismic_service.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
+
+package cognite.seismic.v1;
+
+import "google/protobuf/empty.proto";
+import "cognite/seismic/protos/types.proto";
+import "cognite/seismic/protos/v1/seismic_service_datatypes.proto";
+import "cognite/seismic/protos/v1/seismic_service_messages.proto";
+
+/**
+Service for querying data and metadata from seismic tracestore in Cognite Data Fusion (CDF) seismic datastore.
+
+Queries are done primarily on cubes, with some management endpoints for tracestores & partitions. 
+
+Queries are divided in:
+    Metadata: General information and headers for surveys and files
+    Trace: Retrieve traces, be it by geometry, line or volume
+    Artificial sampling: Get the (calculated/interpolated) values of properties in the file/tracestore given arbitrary positions
+**/
+
+service Seismic {
+// Metadata queries
+
+    /**
+    Get Survey by ID.
+    REMOVED: Use ReadSurveys, providing SearchSpec with the survey ID.
+    rpc GetSurvey (SurveyQueryRequest) returns (GetSurveyResponse) {}
+    **/
+
+    /**
+    Lists all surveys accessible to the client. Optionally includes Cubes associated with the surveys.
+    
+    Can optionally search surveys based on both/either of two criteria:
+        Coverage polygon of files in the survey are within an area delimited by a specified polygon
+        Filters on metadata of both the survey and the file
+    **/
+    rpc ReadSurveys (ReadSurveysRequest) returns (stream ReadSurveysResponse) {}
```

Comment:
According to service contract that endpoint should be either called SearchSurveys (if filtering / search is involved) or GetSurveys (if we are fetching exact ones by id)